admin/class-admin.php

```diff
@@ -112,7 +102,20 @@ public function __construct() {
 		$integrations[] = new WPSEO_MyYoast_Proxy();
 		$integrations[] = new WPSEO_MyYoast_Route();
 		$integrations[] = new WPSEO_Addon_Manager();
-		$integrations[] = $this->admin_features['google_search_console'];
+
+		$this->admin_features = array();
+
+		if ( is_admin() ) {
+			$integrations[] = new WPSEO_GSC();
```

Comment:
I would leave the assigning of the `admin_features` the old way. And then get the value of it by it's key.